apps/data-types.md

```diff
@@ -479,6 +479,8 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | minStocklevel | number | Indicates the minimum stock level of the product. Available if the field *Minimum stock level* is used in the administration area of a shop. Only available with the `products_write` authorisation. |
 | links | array of [link](page:apps-data-types#link) | The links to the product and product category. |
 | productVariationType | string | The type of the variation product. Can be one of *master*, *regular* or *variation*. |
+| productVariationMasterId | string | The unique identifier of the master product of a variation. Null if the product itself is a master or a regular product. |
+| productVariationMasterName | string | The name of the master product of a variation. Null if the product is a master or a regular product. |
```

Comment:
I'm not so happy with the naming of the attribute. Would  it be possible to change it to `variationProductName`?

The name of the variation product. Null if the `productVariationType` is *master* or *regular*.

Reply:
same here

Reply:
ok, but can we still change the description for the datatypes as suggested?
if the attribute renaming would result in a follow-up task, leave it as it is. I thought it could be adjusted at the same time. I don't want to make this an extra effort.